app/src/app/[lng]/[inventory]/data/[step]/[subsector]/page.tsx

```diff
@@ -155,6 +155,22 @@ function SubSectorPage({
     }, 1000);
   };
 
+  const getSectorName = (currentStep: string) => {
+    switch (currentStep) {
+      case "1":
+        return t("stationary-energy");
+      case "2":
+        return t("transportation");
+      case "3":
+        return t("waste");
+      default:
+        return t("stationary-energy");
+    }
+  };
+  const getSubsectorData = useSelector(
```

Comment:
Could be renamed to subSectorData in order to avoid confusion as this is not a function it seems.